ui/src/app/groupings/[groupingPath]/_components/export-dropdown.tsx

```diff
@@ -1,16 +1,129 @@
 'use client';
 
+import { useState, useRef } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faFileExport,
+    faCaretDown,
+    faUsers,
+    faUserPlus,
+    faUserMinus,
+    faIdCard
+} from '@fortawesome/free-solid-svg-icons';
+import { Tooltip, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip';
+import { getGroupingMembers } from '@/lib/actions';
 import { Button } from '@/components/ui/button';
-import { FileInput, ChevronDown } from 'lucide-react';
 
-const ExportDropdown = () => {
+interface Member {
+    lastName: string;
+    firstName: string;
+    uid: string;
+    uhUuid: string;
+}
+
+interface GroupData {
+    members: Member[];
+}
+
+const ExportDropdown = ({ groupPath }: { groupPath: string }) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const dropdownRef = useRef<HTMLDivElement>(null);
+
+    const memberTypes = [
+        { type: '', label: 'All Members', icon: faUsers },
+        { type: 'basis', label: 'Basis', icon: faIdCard },
+        { type: 'include', label: 'Include', icon: faUserPlus },
+        { type: 'exclude', label: 'Exclude', icon: faUserMinus }
+    ];
+
+    const fetchGroupData = async (memberType: string) => {
+        const fullPath = memberType === '' ? groupPath : `${groupPath}:${memberType}`;
+        const params = { page: undefined, size: undefined, sortString: 'name', isAscending: true };
```

Comment:
`page: undefined, size: undefined,` can be removed here because the getGroupingMembers function handles `undefined` params.